README.md

```diff
@@ -110,6 +110,41 @@ possible by the use of the `UserService` service provided by
 You can see the details in
 [`packages/shopping/src/controllers/user.controller.ts`](https://github.com/strongloop/loopback4-example-shopping/blob/master/packages/shopping/src/controllers/user.controller.ts).
 
+### Authorization
+
+For an authorization demo in this example, all requests to the endpoint
+`/users/{userId}/orders` (i.e, `UserOrderController`) are validated for specific
+user access.
+
+Endpoint authorization is done using `@loopback/authorization` package and the
+provisions in it are implemented using
+[Casbin](https://github.com/casbin/casbin) configurations.
+
+The following is a typical scenario for an authorization demo:
```

Comment:
maybe: `Below is a scenario illustrating how authorization works in this example`?

Reply:
agreed